packages/fluent-theme/src/components/activity/CopilotMessageHeader.module.css

```diff
@@ -0,0 +1,42 @@
+
+:global(.webchat-fluent) .copilot-message-header {
+  align-items: center;
+  cursor: default;
+  display: flex;
+  flex-wrap: nowrap;
+  gap: var(--webchat-spacingHorizontalS);
+  margin-inline-start: var(--webchat-spacingVerticalMNudge);
+  padding-block-start: var(--webchat-spacingVerticalXS);
+  /* TODO: remove when moved to the bubble */
+  position: relative;
+  z-index: 1;
```

Comment:
Can we avoid `z-index`?

Reply:
Will remove soon. Without it the header is not selectable when copying text from the chat.